nav2-usability/scenario_code/lib/deeplink-pathparam/deeplink_pathparam_auto_route.dart

```diff
@@ -43,7 +43,7 @@ class BooksApp extends StatelessWidget {
 
     return MaterialApp.router(
       routerDelegate: _appRouter.delegate(),
-      routeInformationParser: _appRouter.defaultRouteParser(),
+      routeInformationParser: _appRouter.defaultRouteParser(includePrefixMatches: true),
```

Comment:
Just curious, can this only apply to a specific route? e.g. I want prefix matches for /books/123, but not for /user/123?

Reply:
I think you would have to use a RouteGuard in that case and push the specific routes you want